src/main/kotlin/car/Vehicle.kt

```diff
@@ -16,14 +16,6 @@ class Vehicle(val motor: Motor){
     }
 
     fun extintor(): Boolean{
-        return motor.useFossilFuel() == true || motor.useFlammelFuel() == true
+            return motor.useFossilFuel() || motor.useFlammelFuel()
```

Comment:
Esse eu deixei passar no outro PR :sweat_smile: 
Só precisava do `useFlammelFuel()`
```suggestion
            return motor.useFlammelFuel()
```